src/GeositeFramework/plugins/launchpad/main.js

```diff
@@ -86,6 +86,33 @@ define([
                 }));
 
                 $(this.container).empty().append($el);
+
+                // add truncate listeners to each toggle botton
+                $(".truncate-container").each(function () {
+                    if (parseFloat($('p', this).css("height")) > 50) {
+                        $('p', this).each(function() {
+                            this.classList.toggle('truncate-box');
+                        });
+
+                        this.addEventListener('click', function(ev) {
+                            if (ev.target.tagName === 'BUTTON') {
+                                $('p', this).each(function() {
+                                    this.classList.toggle('truncate-box');
+                                });
+                                $('.truncate-toggle', this).each(function() {
+                                    if (this.textContent === 'View more') {
+                                        this.textContent = 'View less';
+                                    } else {
+                                        this.textContent = 'View more';
+                                    }
+                                });
+                            }
+                        });
+                    } else {
+                        $(".truncate-toggle", this).css({ display:"none" });
+                    }
```

Comment:
I agree this is more of a design issue. But I would argue that using text length is simpler and looks better. Making sure the "truncate container" height lines up with the text will be an extra chore.

See the difference between these two options:

1. Height
  ![image](https://cloud.githubusercontent.com/assets/43062/21188046/0146628c-c1e8-11e6-96a9-f88eb7960cc3.png)

2. Text length
  ![image](https://cloud.githubusercontent.com/assets/43062/21188053/06828924-c1e8-11e6-94c9-8cb5f4c27427.png)